docs/03-contributing/01-getting_started.md

```diff
@@ -4,6 +4,27 @@ There are several ways to get started. Like using the Docker setup or use the de
 
 The code is mainly written in PHP using the Symfony framework with Twig templating and a bit of JavaScript & CSS and of course HTML.
 
+## Docker as a dev server
+
+To save yourself much time setting up a development server, you can use our Docker setup instead of a manual configuration:
+
+1. Make sure you are currently in your Mbin directory.
+2. Run the auto setup script with `./docker/setup.sh dev localhost` to configure `.env`, `compose.override.yaml`, and `storage/`.
+3. Use `docker build -t mbin -f docker/Dockerfile --target dev .` to build the dev Docker image.
+4. Run `docker compose up` to start the Docker containers.
+5. From here, you should be able to access your server at [https://localhost/](https://localhost/). Any edits to the source files will automatically rebuild your server.
+6. Optionally, follow the [Mbin first setup](../02-admin/04-running-mbin/01-first_setup.md) instructions.
+
+> [!NOTE]
+> If you'd prefer to manually configure your Docker environment (instead of using the setup script) then follow the manual environment setup steps in the [Docker install guide](../02-admin/01-installation/02-docker.md), but while you're creating `compose.override.yaml`, use the following:
+>
+> ```yaml
+> include:
+>   - compose.dev.yaml
+> ```
```

Comment:
We can add maybe another `> [!INFO]` section here about how to stop it again using docker compose down.

Also you might want to explain to use `composer up -d` with the `-d` option. To execute in the background (daemon). Of course this all depends a bit on what kind of knowledge level we expect the admins to have of Docker.

Reply:
I didn't mention the `-d` option because I thought it would be unnecessary in a development environment. You want to be able to see the logs immediately as they come through, and be able to stop the server as needed with `Ctrl+C`.

I could add a note about stopping the server with `Ctrl-C` though. I'm not sure if that's completely necessary though, as the guide doesn't mention stopping the `symfony server:start` command either (as `Ctrl-C` is assumed).

Reply:
> I could add a note about stopping the server with `Ctrl-C` though. I'm not sure if that's completely necessary though, as the guide doesn't mention stopping the `symfony server:start` command either (as `Ctrl-C` is assumed).

That doesn't mean we can't improve the guide :D :D

Reply:
Done: https://github.com/MbinOrg/mbin/pull/1542/commits/b9fdfa4e41cd4b3883dcb64cc1e7eb2910d8bd82!